packages/react-components/src/components/SendBox.tsx

```diff
@@ -126,8 +134,13 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   const [textValue, setTextValue] = useState('');
   const [textValueOverflow, setTextValueOverflow] = useState(false);
 
+  const hasFocusOnMount = props.hasFocusOnMount ?? true;
```

Comment:
default behaviour is focus on SendBox.
Is that what we want? It sounds to be the proper way to go when used in Chat, but just asking